{{cookiecutter.plugin_name}}/{{cookiecutter.module_name}}/_tests/test_reader.py

```diff
@@ -1,10 +1,13 @@
 import numpy as np
+import {{cookiecutter.module_name}}
 from {{cookiecutter.module_name}} import napari_get_reader
 
 
 # tmp_path is a pytest fixture
-def test_reader(tmp_path):
+def test_reader(tmp_path, napari_plugin_manager):
     """An example of how you might test your plugin."""
+    # make_napari_viewer block plugin discovery, so we must register it manually
+    napari_plugin_manager.register({{cookiecutter.module_name}}, name="{{cookiecutter.plugin_name}}")
```

Comment:
since this test doesn't actually go through napari (note that it's calling the `napari_get_reader` function directly...), it's not necessary to register here (